$NetBSD: patch-ac,v 1.4 2017/09/13 08:37:41 wiz Exp $

Don't build tests or the dummy_drv.
dummy_drv isn't useful, and causes PLIST mismatches

--- Makefile.am.orig	2017-05-22 15:15:09.000000000 +0000
+++ Makefile.am
@@ -24,7 +24,7 @@ ACLOCAL_AMFLAGS = -I m4 ${ACLOCAL_FLAGS}
 
 AUTOMAKE_OPTIONS = foreign
 
-SUBDIRS = va dummy_drv_video pkgconfig doc
+SUBDIRS = va pkgconfig doc
 
 # Extra clean files so that maintainer-clean removes *everything*
 MAINTAINERCLEANFILES = \
